src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -182,6 +182,16 @@ public string HubName
         /// </summary>
         public bool UseGracefulShutdown { get; set; } = false;
 
+        /// <summary>
+        /// Whether exceptions inside entity operations cause the state of the entity to be rolled back. The rollback undoes all
+        /// internal effects of an operation (sent signals, and state creation, deletion, or modification).
+        /// However, it does not roll back external effects (such as I/O that was performed).
+        /// If true, the entity state is serialized after each individual operation that accessed the state, to create a snapshot
+        /// from which to restore the state if the next operation fails. If false, the entity state
+        /// is not serialized after each individual operation, but only after an entire batch of operations completes.
+        /// </summary>
```

Comment:
The `<summary>` tag should contain a short, ideally one sentence description of what this property does. The longer explanation should be separated out into a `<remarks>` section.

Reply:
ok.